src/Core/Enums/RuleId.cs

```diff
@@ -126,6 +126,7 @@ public enum RuleId
         ControlShouldSupportSpreadsheetItemPattern,
         ControlShouldSupportTableItemPattern,
         ControlShouldSupportTablePattern,
+        ControlShouldSupportTablePatternInEdge,
```

Comment:
@RobGallo, should we update the RuleDescriptions.md file as rules are modified or when we have a release? I tend to lean towards keeping the code and the description in sync in the repo, independent of the release schedule. Thoughts?

Reply:
I think we should keep them updated as code changes, not on release. I'll add that as a change.